src/docs/release/breaking-changes/mousetrackerannotation-requires-parameter-key.md

```diff
@@ -0,0 +1,177 @@
+---
+title: MouseTrackerAnnotation requires parameter "key"
+description: [Brief description similar to the "context" section below.]
+---
+
+# MouseTrackerAnnotation requires parameter "key"
+
+
+## Context
+
+Mouse events, such as when a mouse pointer has entered a region, exited, or
+is hovering a region, are detected with the help of
+`MouseTrackerAnnotation`s that are placed on interested regions during the
+render phase. Upon each new frame or new event, `MouseTracker` compares the
+annotations hovered by the mouse pointer before and after the change, then
+dispatches callbacks accordingly. `MouseTracker` also tracks which
+annotations are on the screen by requring the annotation's owners to attach
+or detach the annotation promptly.
+
+However, since the `MouseTrackerAnnotation` is a constant class, its properties
+can't be changed without replacing the annotation as a whole, which triggers
+the exit and enter callback if it's hovered by a mouse pointer.
+
+This issue is a blocker to the plan of supporting mouse cursor, since it’s
+very likely that the cursor assigned to a region needs to be changed.
```

Comment:
We should write this from the point of view of having already made the change. There used to be a problem, the problem was bla; this change resolves that issue.

Reply:
Fixed.